.gitignore

```diff
@@ -1 +1,2 @@
 *.pyc
+.idea/
```

Comment:
OMZ itself doesn't produce this folder. This should be in your personal global .gitignore